### DISCUSSION TITLE
Error logs have regressed to the point they are basically useless.

### DISCUSSION THREAD

**Discussion by StylezDevops** [microsoft/powerplatform-build-tools#341] [Ideas]
The previous power platform build tasks used to show detailed error logs. The current ones display something akin to this:
<img width="1258" alt="image" src="https://user-images.githubusercontent.com/94297031/206134020-e8431127-fbd7-4eb5-9a7e-343b9889e6b5.png">

In order to get to the actual REASON for the failed solution import I then have to navigate to make.powerapps.com and view the history

<img width="299" alt="image" src="https://user-images.githubusercontent.com/94297031/206136705-e0bf4931-6610-452c-8ecd-51f96f667fa4.png">

This is the information we need to see in the Task logs. This has been raised but was raised on #239  but that was another issue which then got closed. 

I would like to see a return to how the previous build tasks showed the logs. Thanks.

**Comment by cT-m00cat** [comment]
ping

**Comment by joem-msft** [comment]
Have you tried clicking on the "Enable system diagnostics" when running your pipeline?
The log files should then contain more detailed logs coming from the internal Pac CLI.

**Comment by cT-m00cat** [comment]
Thanks... This is a good tip and helps somewhat in the immediate future - however, the previous version of the tasks had detailed logging and output this as standard. The removal of this is a regression. For this reason I no longer user the build tools tasks and do everything in a powershell task with pac cli directly.

**Comment by StylezDevops** [comment]
Yo! @cT-m00cat, this situation is straight-up regression. I gotta remember to enable system diagnostics, 'cause that's the fix, you know what I'm saying? But honestly, I'd rather have the tasks go back to how they used to be, showing all the detailed log output like before. Might have to take a page outta @cT-m00cat's book and just use pac cli directly.
